### PR TITLE
[DynamicDelivery] Improve fullsize gradle

### PR DESCRIPTION
**Pull request by Mercandj**

### CODE REVIEW

File: mwm-5-dynamic-delivery/presentation/PITCHME.md [hunk 1]
```diff
@@ -236,6 +236,61 @@ dependencies {
...
+        debug {
+            shrinkResources false
+        }
+    }
+
+    dynamicFeatures = [":app_search_dynamic"]
+}
+
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
```
Comment (bowserf) [inline mwm-5-dynamic-delivery/presentation/PITCHME.md]: Noise

Comment (Mercandj) [reply mwm-5-dynamic-delivery/presentation/PITCHME.md]: Sure

Comment (Mercandj) [reply mwm-5-dynamic-delivery/presentation/PITCHME.md]: Done

**Comment by Mercandj** [comment]
thks

**Pull request merged**